l10n_br_stock_account/models/stock_picking.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (C) 2014  Renato Lima - Akretion
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import api, fields, models
+
+
+class StockPicking(models.Model):
+    _name = 'stock.picking'
+    _inherit = [_name, 'l10n_br_fiscal.document.mixin']
+
+    @api.model
+    def _default_fiscal_operation(self):
+        company = self.env.user.company_id
+        fiscal_operation = self.env.user.company_id.stock_fiscal_operation_id
+        picking_type_id = self.env.context.get('default_picking_type_id')
+        if picking_type_id:
+            picking_type = self.env['stock.picking.type'].browse(
+                picking_type_id)
+            fiscal_operation = picking_type.fiscal_operation_id or (
+                company.stock_in_fiscal_operation_id
+                if picking_type.code == 'incoming'
+                else company.stock_out_fiscal_operation_id
+            )
+        return fiscal_operation
```

Comment:
Isto não esta considerando o terceiro campo do cadastro da empresa

Reply:
Está considerando os três campos: se não houver um picking_type definido por padrão ele retorna o conteúdo do campo "Stock Fiscal Operation"; se houver um picking_type definido por padrão e ele for do tipo 'incoming', a função retorna o conteúdo do campo "Stock In Fiscal Operation"; e se houver um picking_type definido por padrão e ele for do tipo 'outcoming', a função retorna o conteúdo do campo "Stock Out Fiscal Operation". Esses três campos são os definidos na aba Estoque da seção Fiscal do cadastro da empresa. A lógica de prioridade dos campos não é essa?

Reply:
É essa sim, mas veja que o cadastro da empresa tem um terceiro campo e ele não esta sendo usado.